tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/client/DeadLetterAddressTest.java

```diff
@@ -82,6 +87,43 @@ public void testBasicSend() throws Exception {
       Assert.assertEquals(m.getBodyBuffer().readString(), "heyho!");
    }
 
+   @Test
+   public void testLargeMessageFileLeak() throws Exception {
+      // only run this on *nix systems which will have the com.sun.management.UnixOperatingSystemMXBean (needed to check open file count)
+      Assume.assumeTrue(File.separatorChar == '/');
```

Comment:
I tested it on windows and it does compile file...


Can you change this to some sort of instanceof?